Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -21,15 +21,24 @@ struct CreateAndEditGeometriesView: View {
     @State private var map = Map(basemapStyle: .arcGISTopographic)
     
     /// The model that is required by the menu.
-    @StateObject var model = GeometryEditorMenuModel(
-        geometryEditor: GeometryEditor(),
-        graphicsOverlay: GraphicsOverlay(renderingMode: .dynamic)
-    )
+    @StateObject var model = GeometryEditorMenuModel()
     
     var body: some View {
         VStack {
             MapView(map: map, graphicsOverlays: [model.graphicsOverlay])
                 .geometryEditor(model.geometryEditor)
+                .task {
+                    for await geometry in model.geometryEditor.$geometry {
+                        // Update geometry when there is an update.
+                        model.onGeometryChanged(geometry)
+                    }
+                }
+                .task {
+                    for await selection in model.geometryEditor.$selectedElement {
```

Comment:
A better name for the loop variable would be `selectedElement` or perhaps even just `element`.